charts/seed-controlplane/charts/kube-scheduler/templates/componentconfig.yaml

```diff
@@ -7,7 +7,11 @@ metadata:
 data:
   config.yaml: |-
     ---
-    apiVersion: {{ include "schedulercomponentconfigversion" . }}
+    apiVersion: {{ if semverCompare ">= 1.12" .Values.kubernetesVersion -}}
```

Comment:
Better put this in a template definition and include it.

Reply:
Agreed, please do it similar to how it was done for the kube-apiserver's audit config version.